keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -188,16 +193,16 @@ This also falls inline with how in-place update of Pod is handled(https://github
 
 * **Does enabling the feature change any default behavior?**
   Allow users to reduce size of pvc in `pvc.spec.resources`. In general this was not permitted before
-  so it should not break any of existing automation. This means that if `pvc.Spec.AllocatedResources` is available it will be
+  so it should not break any of existing automation. This means that if `pvc.Status.AllocatedResources` is available it will be
   used for calculating quota.
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
```

Comment:
after I disable this feature, I think I'll have mismatched spec and status on PVCs.  What will happen?

Reply:
You are only allowed to reduce `pvc.Spec.Resources` upto value in `pvc.Status.Capacity` when feature gate is enabled, so if feature gate is disabled then for in-flight expansions, they will be allowed to complete (even when feature gate is disabled).  resize controller will try expansion as long as `pvc.Spec.Resources` is smaller than `pv.Spec.Capacity` for control-plane expansion and `pvc.Status.Capacity` is smaller than `pv.Spec.Capacity`. In a nutshell - the spec and status should not be mismatched.